deploy/roles/connect/templates/cloudwatch-config.json.j2

```diff
@@ -3,6 +3,20 @@
         "metrics_collection_interval": 60,
         "run_as_user": "cwagent"
     },
+    "logs": {
+        "logs_collected": {
+            "files": {
+                "collect_list": [
+                    {
+                        "file_path": "/var/log/syslog",
+                        "log_group_name": "{{ deploy_environment }}_syslog.log",
+                        "log_stream_name": "{{ deploy_environment }}_syslog.log",
```

Comment:
Does this need some way to make sure different instances logs don't get mixed? It looks like for HQ, the log stream name includes the instance id and instance name within the same log group, to separate them. I think if we don't do that, the syslogs of all instances will mix. It appears we can get that info using special variables `{instance_id}` and/or `{hostname}`